src/controllers/user.controller.js

```diff
@@ -0,0 +1,49 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService
+    .getAll()
+    .then((usrs) => usrs.map(userService.normalize));
```

Comment:
The use of both `async/await` and `.then()` for handling promises in the `get` function is inconsistent. Consider using `await` for the entire promise chain for better readability and consistency.